src/main/webapp/app/shared/select/CancerTypeSelect.tsx

```diff
@@ -24,7 +24,7 @@ export type CancerTypeSelectOption = {
 };
 
 const getAllMainTypes = (cancerTypeList: ICancerType[]) => {
-  return _.uniq(cancerTypeList.filter(cancerType => cancerType.level <= 0)).sort();
+  return _.uniq(cancerTypeList.filter(cancerType => cancerType.level && cancerType.level <= 0)).sort();
```

Comment:
Here is an example of something that cannot be null (according to our database rules). As Calvin mentioned, a better solution would be to fix our models.